REVIEW.md

```diff
@@ -0,0 +1,192 @@
+# Reviewer Guidelines
+
+Reviewing pull requests (PRs) can take many forms and be reflected across many styles.
+To help make the process more clear for contributors, the following guidelines are designed to scope review and specify what areas of feedback block PRs from merging.
+
+**Note:** These guidelines are new; feedback on them is appreciated and can be given via [issue to the main spyder repo](https://github.com/spyder-ide/spyder/issues).
```

Comment:
```suggestion
**Note:** These guidelines are new; feedback on them is appreciated and can be given via [opening an issue on the main Spyder repo](https://github.com/spyder-ide/spyder/issues/new/choose).
```

This change got lost in the merge (which I realized after I got a sense of deja vu making it, heh), so I made it again